docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-version: '3.8'
+version: "3.8"
 
 services:
   web:
     build: .
-    command: [ "./entrypoint.sh" ]
+    command: ["./entrypoint.sh"]
     volumes:
       - .:/app
```

Comment:
To avoid polluting the workspace with artifacts generated in the container (e.g. data.json downloaded in entrypoint.sh), which might lead to false positives, we should remove this for now.
```suggestion
```